pages/app-layout/utils/external-widget.tsx

```diff
@@ -165,15 +166,15 @@ awsuiPlugins.appLayout.registerDrawer({
 });
 
 awsuiPlugins.appLayout.registerDrawer({
-  id: 'circle2-global',
+  id: 'global-with-stored-state',
   type: 'global',
   defaultActive: false,
   resizable: true,
   defaultSize: 320,
 
   ariaLabels: {
     closeButton: 'Close button',
-    content: 'Content',
+    content: 'Drawer with counter',
```

Comment:
Improvement for manual testing. All drawers have the same circle icons, let's have different tooltips at least